src/axom/mainpage.md

```diff
@@ -19,8 +20,9 @@ Dependencies between components are as follows:
 - Core has no dependencies, and the other modules depend on Core
 - Slic optionally depends on Lumberjack
 - Slam, Primal, Mint, Quest, Spin, and Sidre depend on Slic
+- Inlet depends on Sidre, and Slic
 - Mint optionally depends on Sidre
-- Spin depends on Primal and Slam
+- Spin depends on Primal, and Slam
 - Quest depends on Slam, Primal, Spin, and Mint
 
 The figure below summarizes the dependencies between the modules.  Solid links
```

Comment:
As far as I can tell.  I also tried moving one (lumberjack's) to under the docs/doxygen directory and then it could no longer find it even though I changed the path in the Doxyfile.in.